frontend/views/Landing.vue

```diff
@@ -232,6 +233,7 @@
         <router-link
           to="/home"
           tag="button"
+          alt="cooperation"
```

Comment:
Same as above if the element is only decorative.

Reply:
Same as above.